console/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "lib": ["es6", "dom", "es2017"],
+    "checkJs": true,
+    "allowJs": true,
+    "jsx": "react",
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "esModuleInterop": true,
+    "noEmitOnError": false
+  },
```

Comment:
Should we also do `noExplicitAny`?

Reply:
Sometimes it's okay to use `any`. Example:
```js
declare const log: <T>(x: T) => void;
```
The function above is not generic, `T` is not used anywhere else than as the type of `x`, so it can as well be `any` (which is more accurate since we definitely don't care about this type).